lib/tasks/one_off.rake

```diff
@@ -257,4 +257,43 @@ namespace :one_off do
     puts "Total packages checked: #{input_tsv_file.count}"
     puts "Versions corrected: #{corrected_count}"
   end
+
+  desc "Retrieve Maven versions from ignored sources"
+  task :retrieve_maven_versions_from_ignored_sources, %i[output_file] => :environment do |_t, args|
+    maven_versions = Version.where(project_id: Project.where(platform: "Maven"))
+    # We want to retrieve the versions where the repository_sources are not solely ["Maven"] or ["Google"]
+    ignored_maven_versions = maven_versions.where.not("repository_sources = ?", ["Maven"].to_json).where.not("repository_sources = ?", ["Google"].to_json)
```

Comment:
Is this using an `OR` in the query? If not, does this produce an `OR`?

```suggestion
    ignored_maven_versions = maven_versions.where.not("repository_sources = ?", ["Maven"].to_json).or(maven_versions.where.not("repository_sources = ?", ["Google"].to_json))
```

Reply:
We discussed this and determined that changing this query changes the results to something we do not desire, so leaving it as-is.